homeassistant/components/conversation/llm_task.py

```diff
@@ -0,0 +1,48 @@
+"""LLM tasks to be handled by conversation agents."""
+
+from dataclasses import dataclass
+from enum import StrEnum
+from typing import Any
+
+
+class LLMTaskType(StrEnum):
+    """LLM task types."""
+
+    GENERATE = "generate"
+    SUMMARIZE = "summarize"
```

Comment:
What do you think about guidance like this:

```suggestion
    GENERATE = "generate"
    """Generate content, which may target a higher quality result."""
    
    SUMMARIZE = "summarize"
    """Summarize existing content, which be able to use a more cost effective model."""
```